core/templates/pages/learner-dashboard-page/content-toggle-button/content-toggle-button.component.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for content toggle button
+ */
+import {Component, EventEmitter, Output} from '@angular/core';
+import {downgradeComponent} from '@angular/upgrade/static';
+import {TranslateService} from '@ngx-translate/core';
+@Component({
+  selector: 'oppia-content-toggle-button',
+  templateUrl: './content-toggle-button.component.html',
+})
+export class ContentToggleButtonComponent {
```

Comment:
I think the name is very unclear, what content exactly is this button supposed to toggle? Can we make the name more clear accordingly/

Reply:
I used a more generic name since it was going to be used in the progress and goal tabs. Would it be better to name it like toggleGoalsandLessons? 


<img width="946" alt="Screenshot 2024-09-02 at 11 45 19 AM" src="https://github.com/user-attachments/assets/b2d28e9f-96a0-4f99-978f-3a1177c8a841">


<img width="946" alt="Screenshot 2024-09-02 at 11 45 03 AM" src="https://github.com/user-attachments/assets/cf71f917-8a87-40f9-be71-993b697efabb">

Reply:
Ah actually never mind, I think the name is fine since it's inside the learner dashboard page directory.